mmdet3d/datasets/seg3d_dataset.py

```diff
@@ -107,14 +107,15 @@ def __init__(self,
         metainfo['palette'] = updated_palette
 
         # construct seg_label_mapping for semantic mask
-        seg_max_cat_id = len(self.METAINFO['seg_all_class_ids'])
-        seg_valid_cat_ids = self.METAINFO['seg_valid_class_ids']
-        neg_label = len(seg_valid_cat_ids)
-        seg_label_mapping = np.ones(
-            seg_max_cat_id + 1, dtype=np.int) * neg_label
-        for cls_idx, cat_id in enumerate(seg_valid_cat_ids):
-            seg_label_mapping[cat_id] = cls_idx
-        self.seg_label_mapping = seg_label_mapping
+        if not hasattr(self, 'seg_label_mapping'):
```

Comment:
We should implement a new function named `build_seg_label_mapping`, and call it in this place.
So that semantickitti_dataset could override this function and we could skip the check here

Reply:
I've fixed it.